specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-01-01/machineLearningServices.json

```diff
@@ -0,0 +1,6522 @@
+      }
+    },
+    "RegistryListCredentialsResult": {
+      "type": "object",
+      "properties": {
+        "location": {
+          "readOnly": true,
+          "type": "string"
+        },
+        "username": {
+          "readOnly": true,
+          "type": "string"
+        },
+        "passwords": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Password"
+          }
+        }
+      }
+    },
+    "Password": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "readOnly": true,
+          "type": "string"
+        },
+        "value": {
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "PaginatedComputeResourcesList": {
+      "type": "object",
+      "description": "Paginated list of Machine Learning compute objects wrapped in ARM resource envelope.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "An array of Machine Learning compute objects wrapped in ARM resource envelope.",
+          "items": {
+            "$ref": "#/definitions/ComputeResource"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "A continuation link (absolute URI) to the next page of results in the list."
+        }
+      }
+    },
+    "ComputeResource": {
+      "type": "object",
+      "description": "Machine Learning compute object wrapped into ARM resource envelope.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "properties": {
+              "description": "Compute properties",
+              "$ref": "#/definitions/Compute"
+            }
+          }
+        }
+      ]
+    },
+    "Compute": {
+      "type": "object",
+      "description": "Machine Learning compute object.",
+      "discriminator": "computeType",
+      "properties": {
+        "computeType": {
+          "description": "The type of compute",
+          "$ref": "#/definitions/ComputeType"
+        },
+        "computeLocation": {
+          "description": "Location for the underlying compute",
+          "type": "string"
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "The provision state of the cluster. Valid values are Unknown, Updating, Provisioning, Succeeded, and Failed.",
+          "enum": [
+            "Unknown",
+            "Updating",
+            "Creating",
+            "Deleting",
+            "Succeeded",
+            "Failed",
+            "Canceled"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          },
+          "readOnly": true
+        },
+        "description": {
+          "type": "string",
+          "description": "The description of the Machine Learning compute."
+        },
+        "createdOn": {
```

Comment:
You should remove this, since it is already covered by the systemData.createdAt. Same goes for modifiedOn, which is covered by lastModifiedAt in systemData.

Reply:
+ @joshanator , I removed these two properties, please correct the service as needed thanks.

Reply:
I believe we still need to keep them in the computeEntity contract for our backend. But Removing in swagger should be fine as it is covered in systemData